components/profile/ProfileDetail.vue

```diff
@@ -349,6 +367,72 @@ useAsyncData('tabs-count', async () => {
   }
 })
 
+const fetchTabsCount = async (chain: Prefix) => {
```

Comment:
add more query on profile?
consider refactor since +100 LOC

Reply:
It would fetch asset data from different network on profile page.
This function has only 40 LOC. There is no `big` function defined in this file.